freeipa/src/main/java/com/sequenceiq/freeipa/flow/stack/stop/StackStopService.java

```diff
@@ -12,6 +12,7 @@
 import com.sequenceiq.freeipa.flow.stack.StackFailureEvent;
```

Comment:
nit: the commit message line length is pretty long, can you add some new lines?

Reply:
Thx! Updated.